<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="SZ-00343555-01" failures="2" tests="11" name="Dong&#039;s测试模块" time="5.139" errors="0" timestamp="12 8月 2022 02:51:20 GMT">
  <testcase classname="testcase.TestDemo" name="checkMapNotNull" time="1.041"/>
  <testcase classname="testcase.TestDemo" name="test" time="0.001"/>
  <testcase classname="testcase.TestDemo" name="test" time="0.002"/>
  <testcase classname="testcase.TestDemo" name="test" time="0.003"/>
  <testcase classname="testcase.TestDemo" name="test1" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
	at testcase.TestDemo.test1(TestDemo.java:20)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="testcase.TestDemo" name="testHttpGet" time="0.698"/>
  <testcase classname="testcase.TestDemo" name="testHttpPost" time="0.044"/>
  <testcase classname="testcase.TestDemo" name="testHttpPostMap" time="0.005"/>
  <testcase classname="testcase.TestDemo" name="testTimeOut" time="3.024">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method testcase.TestDemo.testTimeOut() didn&amp;apos;t finish within the time-out 3000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method testcase.TestDemo.testTimeOut() didn't finish within the time-out 3000
	at java.base@17.0.3.1/java.util.Arrays.equals(Arrays.java:2978)
	at java.base@17.0.3.1/java.lang.invoke.MethodType.equals(MethodType.java:858)
	at java.base@17.0.3.1/java.lang.invoke.MethodType.equals(MethodType.java:850)
	at java.base@17.0.3.1/java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:940)
	at java.base@17.0.3.1/java.lang.invoke.MethodType$ConcurrentWeakInternSet.get(MethodType.java:1368)
	at java.base@17.0.3.1/java.lang.invoke.MethodType.makeImpl(MethodType.java:341)
	at java.base@17.0.3.1/java.lang.invoke.MethodTypeForm.canonicalize(MethodTypeForm.java:253)
	at java.base@17.0.3.1/java.lang.invoke.MethodTypeForm.findForm(MethodTypeForm.java:220)
	at java.base@17.0.3.1/java.lang.invoke.MethodType.makeImpl(MethodType.java:356)
	at java.base@17.0.3.1/java.lang.invoke.MethodHandleNatives.findMethodHandleType(MethodHandleNatives.java:399)
	at java.base@17.0.3.1/java.util.concurrent.FutureTask.set(FutureTask.java:229)
	at java.base@17.0.3.1/java.util.concurrent.FutureTask.run(FutureTask.java:272)
	at java.base@17.0.3.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base@17.0.3.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@17.0.3.1/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- testTimeOut -->
  <testcase classname="testcase.TestEmail" name="testSendMail" time="0.074"/>
  <testcase classname="testcase.TestFileUtil" name="testFileUtils" time="0.0">
    <skipped/>
  </testcase> <!-- testFileUtils -->
  <testcase classname="testcase.TestDemo" name="testOkHttpGet" time="0.0">
    <ignored/>
  </testcase> <!-- testOkHttpGet -->
</testsuite> <!-- Dong's测试模块 -->
